mora/service/orgunit.py

```diff
@@ -446,7 +446,7 @@ def create_org_unit():
     org_uuid = organisationenhed_get['relationer']['tilhoerer'][0]['uuid']
     org_unit_type_uuid = req.get(keys.ORG_UNIT_TYPE).get('uuid')
     addresses = [
-        address.get_relation_for(addr)
+        address.get_relation_for(addr, req)
```

Comment:
Why is `req` used as fallback here? Is it to get the validity from the request?

Reply:
> Why is `req` used as fallback here? Is it to get the validity from the request?

Correct.

Reply:
I am really not a fan of this. So far, the `fallback` has been used to supply alternate address objects to the function as a fallback, so to now use it for an arbitrary request object that just so happens to have some properties we need, seems like an abuse of the parameter more than anything.

Reply:
I get what you mean, but the use is not _entirely_ inconsistent: Whereas we'd previously get just address & type from the object itself, now we also get address, type and validity.

The main downside, of course, is that it might allow some truly weird requests to succeed in an unexpected manner, where the value gets picked up from the request itself rather than the addresses.

Coming to think of it, this code might not be necessary; I'll check…